lib/rom/setup/finalize.rb

```diff
@@ -71,19 +63,18 @@ def load_relations(schema)
       end
 
       # @api private
-      def build_relation(name, builder, block = nil)
-        adapter = adapter_relation_map[name]
+      def build_relation(name, builder, options = {}, block = nil)
+        repo_name = options.fetch(:repository) { :default }
```

Comment:
We should document the `:default` repo functionality somewhere. It has already tripped a few people up when new to rom.